docs/installation/installation.md

```diff
@@ -43,52 +43,49 @@ I1216 07:37:45.862959    4256 cert.go:230] Generate ca certificate success.
-|  Notice:                                                                                                                                                                     |
-|    Cross-network, need to change the config server address.                                                                                                                  |
-|                                                                                                                                                                              |
-| (In member kubernetes)~#  kubectl create ns karmada-system                                                                                                                   |
-| (In member kubernetes)~#  kubectl create secret generic karmada-kubeconfig --from-file=karmada-kubeconfig=/root/karmada-apiserver.config  -n karmada-system                  |
-|                                                                                                                                                                              |
-| Step 3: Create karmada agent                                                                                                                                                 |
-|                                                                                                                                                                              |
-| (In member kubernetes)~#  MEMBER_CLUSTER_NAME="demo"                                                                                                                         |
-| (In member kubernetes)~#  sed -i "s/{member_cluster_name}/${MEMBER_CLUSTER_NAME}/g" karmada-agent.yaml                                                                       |
-| (In member kubernetes)~#  kubectl create -f karmada-agent.yaml                                                                                                               |
-|                                                                                                                                                                              |
-| Step 4: Show members of karmada                                                                                                                                              |
-|                                                                                                                                                                              |
-| (In karmada)~# kubectl  --kubeconfig /etc/karmada/karmada-apiserver.config get clusters                                                                                      |
-└──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+------------------------------------------------------------------------------------------------------
```

Comment:
Looks like the other changes with this file have been synchronized.

Reply:
Yes. Here has conflict. I use the new one

Reply:
I think i should use the old version and change create to modify. Is this will be better

Reply:
Does this output fit the `release-1.1`?
I mean the output may be different between the release branch and master, and documents in the release branch should fit the behavior in that release.

Reply:
Let  me check it

Reply:
They are not the same, for example
In master
`Step 1: Use karmadactl join to register the cluster to Karmada control panel. --cluster-kubeconfig is members kubeconfig.`
In release 1.1
` Use kubectl karmada join to register the cluster to Karmada control panel. --cluster-kubeconfig is members kubeconfig.`
I think I should close this PR

Reply:
> I think I should close this PR

Close this and reopen a new one is an option.

But it's not complicated to update this PR, you can do that by following commands:
- 1. fetch the branch(`automated-cherry-pick-of-#1604-origin-release-1.1`) to your workspace. 
- 2. Make an update to it
- 3. git commit --amend
- 4. git push origin automated-cherry-pick-of-#1604-origin-release-1.1 -f

Reply:
Thanks. I use `utils.GenExamples("/etc/karmada","kubectl karmada")` output  replace the conflict part

Reply:
That's fine.